cfe_internal/update/update_policy.cf

```diff
@@ -308,10 +312,33 @@ bundle agent cfe_internal_update_policy_cpv
         usebundle => cfe_internal_setup_python_symlink("$(python_symlink)");
 }
 
+bundle agent cfe_internal_update_cmdb
+# @brief Ensure local cache of CMDB data is up to date
+{
+  files:
+      "$(sys.workdir)/data/." create => "true";
+
+      "$(sys.workdir)/data/host_specific.json" -> { "ENT-6788" }
```

Comment:
@olehermanse, @vpodzime I still think this file name could be better.

`host_specific.json` doesn't really describe the data provenance.

In another place we discussed integrating other data sources, and that might end up named  something like servicenow.json. servicenow.json

Reply:
> host_specific.json doesn't really describe the data provenance.

And?

This is the most straight forward and user friendly name I can think of. It's data. It's host specific. I think being easy to remember and easy to understand, especially for new users, is important.

All individual variables should have descriptive tags of where they came from, not necessarily encoded in the filename.

In the future we might have other files based on classes, like `ubuntu.json`, `windows.json`, `policy_server.json`, based on integrations, like `servicenow.json` or potentially other groupings / sources (class expressions? mission portal groups? regexes matching class names? Variable conditions? Maybe we want to add information about who set the data? And how they did it? (API, MP, integration, CLI?)

Trying to encode all of that in the file name is not a good idea IMO.

For me, as a user, seeing a directory with:

```
/var/cfengine/data
 - host_specific.json
 - ubuntu.json
 - webservers.json
```

Is awesome. I don't need to ask anyone about abbreviations, or anything else. It's self explanatory.

I then open the files, and see something like:

```json
{
  "vars": {
    "roles": {
      "value": ["cfengine_client", "webserver", "production"],
      "mission_portal_title": "Host Roles",
      "description": "The different roles a machine fulfills, including it's cfengine role, application purposes and environment(s)",
      "from": {
        "interface": "Mission Portal",
        "using": "GUI",
        "at": "2021-03-09T20:16:00Z",
        "by": "Nick Anderson"
      }
    }
  }
}
```

(That's just an example, and not the correct structure or keys, but I think it illustrates my point).